prowler/providers/aws/services/kms/kms_cmk_not_deleted_unintentionally/kms_cmk_not_deleted_unintentionally.py

```diff
@@ -0,0 +1,27 @@
+from prowler.lib.check.models import Check, Check_Report_AWS
+from prowler.providers.aws.services.kms.kms_client import kms_client
+
+
+class kms_cmk_not_deleted_unintentionally(Check):
+    def execute(self):
+        findings = []
+        for key in kms_client.keys:
+            if key.manager == "CUSTOMER":
+                report = Check_Report_AWS(self.metadata())
+                report.region = key.region
+                report.resource_tags = key.tags
+                report.status = "PASS"
+                report.status_extended = (
+                    f"KMS CMK {key.id} is not used and is not scheduled for deletion."
+                )
+                report.resource_id = key.id
+                report.resource_arn = key.arn
+                if key.state == "PendingDeletion":
+                    report.status = "FAIL"
+                    report.status_extended = (
+                        f"KMS CMK {key.id} is scheduled for deletion."
+                    )
+                elif key.state == "Enabled":
+                    report.status_extended = f"KMS CMK {key.id} is being used."
```

Comment:
I meant by putting at the beginning of the loop:
`if key.state == "Enabled" or kms_client.provider.scan_unused_services`

Reply:
And adding a test for the case of an unused key

Reply:
Done here https://github.com/prowler-cloud/prowler/pull/4595/commits/a1316eaa33b1ded3062bdf30afc7d49886cd3c17